modules/cluster-logging-logstore-limits.adoc

```diff
@@ -54,21 +56,25 @@ spec:
 ----
 <1> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank,
 the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `16Gi` for the memory request and `1` for the CPU request.
-<2> Specify the CPU and memory requests for the Elasticsearch proxy as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `256Mi` for the memory request and `100m` for the CPU request.
+<2> The maximum amount of resources a Pod may use.
+<3> The minimum resources required to schedule a Pod.
```

Comment:
There should be "pod" I think, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references